front-end/src/components/Signin.js

```diff
@@ -0,0 +1,239 @@
+                                </div> :
+                                <Otp otp={this.state.otp} setOtp={val => this.setState({ otp: val })} />
+                        }
+                        {
+                            this.state.otpShow ?
+                                <div style={{ width: '100%', display: 'flex', justifyContent: 'flex-end', alignItems: 'center', marginTop: 5 }}>
+                                    Didn't receive an OTP?
+                                    <Button
+                                        variant="contained"
+                                        disabled={this.state.time.s > 0}
+                                        style={{
+                                            color: 'white',
+                                            marginLeft: 'auto',
+                                            textTransform: 'none'
+                                        }}
+                                        color="primary"
+                                        onClick={() => this._getCode()}>
+                                        Resend
+                                    </Button>
+                                    <div style={{ marginLeft: 5 }}>
+                                        {this.state.time.s}s
+                                    </div>
+                                </div> :
+                                null
+                        }
+                        <div style={{ display: 'flex', flexDirection: 'row', marginTop: 20 }}>
+                            <Button
+                                variant="contained"
+                                disabled={(this.state.pno.length !== 10) || (this.state.code === null) || !isNumeric(this.state.pno) || (this.state.otpShow && this.state.otp.length !== 6)}
+                                color="secondary"
+                                style={{
+                                    color: 'white',
+                                    marginLeft: 'auto',
+                                    textTransform: 'none'
+                                }}
+                                onClick={() => {
+                                    if (this.state.otpShow) {
+                                        // this.startTimer();
+                                        this._verifyCode();
+                                    } else {
+                                        this._getCode();
+                                        this.startTimer();
+                                        this.setState({ otpShow: true });
+                                    }
+                                }}>
+                                Verify
+                            </Button>
+                        </div>
+                        {
+                            !this.state.otpShow ?
+                                <p>By tapping Verify an SMS may be sent. Message & data rates may apply.</p> :
+                                null
```

Comment:
This message is displayed to the Slowvid user "Message & data rates may apply." Are there plans where it costs money to receive SMS? I suggest remove or reword to indicate it's about the receiving of SMS.